docs/source/upgrade-guide/index.md

```diff
@@ -15,6 +15,44 @@ This upgrade guide lists all breaking changes in Volto and explains the
 
 ## Upgrading to Volto 14.x.x
 
+### Update i18n configuration for projects and addons
+
+The i18n script and infrastructure has been moved to its own package, since we needed it
+to be independent of Volto itself. This was necessary for being able to use it from the
+addons without having to install whole Volto with them (which is not possible).
+
+`@plone/scripts` package is the placeholder of the script, which has been also improved
+and the infrastructure (Babel config) for it to run.
+
+Steps for migration:
+
+In a project, `package.json` replace the `scripts` `i18n` line with this one:
+
+```diff
+   "scripts": {
+-    "i18n": "NODE_ENV=production node src/i18n.js"
++    "i18n": "rm -rf build/messages && NODE_ENV=production i18n --addon"
++  },
+```
+
+and add this to the `dependencies` list:
+
+```diff
++  "dependencies": {
++    "@plone/scripts": "latest"
```

Comment:
maybe it should be "@plone/scripts": "*" ?

Reply:
I was hesitant... even tempted to pin it. But then we might want an updater script...

Reply:
Is there a way to mark "whatever one you have already inherited from the environment"? Yesterday trying, * it just got the latest one... :(

Reply:
I think it's not an issue if you get the latest one, but I'm thinking that maybe someone, at one day, will want to pin it at their project level.